test/tasks/NestedMultisigTask.t.sol

```diff
@@ -301,7 +319,17 @@ contract NestedMultisigTaskTest is Test {
 
         // execute the task
         multisigTask = new TestOPCMUpgradeVxyz();
+
+        /// snapshot before running the task so we can roll back to this pre-state
+        uint256 newSnapshot = vm.snapshot();
+
         multisigTask.simulateRun(opcmTaskConfigFilePath);
+        bytes32 taskHash = multisigTask.getHash();
+
+        /// now run the executeRun flow
+        vm.revertTo(newSnapshot);
+
+        multisigTask.executeRun(opcmTaskConfigFilePath, prepareSignatures(multisig, taskHash));
```

Comment:
What does snapshot and reverting do for us here? It seems we aren't asserting that anything is equal between the `simulateRun` and `executeRun` flows?

Reply:
> We are testing the `executeRun` method by running it.

So we are just testing that it does not revert? Do we have stronger assertions somewhere, e.g. that the `accountAccesses` returned by `simulateRun` are equivalent to those returned by `executeRun` (with the exception of potential differences resulting from e.g. state overrides)

Reply:
Created https://github.com/ethereum-optimism/superchain-ops/issues/637 and moved to the bottom of https://github.com/ethereum-optimism/superchain-ops/issues/396, @blmalone feel free to reorganize where that new issue lives in the tracker